README.md

```diff
@@ -1,2 +1,140 @@
+
+# Math-magician
+> "Math magicians" is basically a website for everyone who loves mathematics. It is not much cause its a Single Page App that enables users to make simple calculations and read a random math-related quote. 
+
```

Comment:
- [ ] [OPTIONAL] I think adding a screenshot of the project will be a good idea. It will make your Readme more descriptive.

Reply:
I added a screenshot